packages/react-codemirror/src/lang-cypher/createCypherTheme.ts

```diff
@@ -61,8 +61,9 @@ export const createCypherTheme = ({
       color: settings.gutterForeground,
       border: 'none',
     },
-    '&.cm-editor .cm-scroller': {
+    '&.cm-editor': {
       fontFamily: 'Fira Code, Menlo, Monaco, Lucida Console, monospace',
+      height: '100%',
```

Comment:
This change is so that consumers can set a minimum height via the `className` property and have the background color extend to the full height